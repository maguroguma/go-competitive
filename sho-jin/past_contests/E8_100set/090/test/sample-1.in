0 2
6 3
2 4
