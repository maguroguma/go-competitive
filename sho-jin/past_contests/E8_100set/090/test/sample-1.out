2.061552812808830
